charts/airflow/docs/faq/configuration/airflow-version.md

```diff
@@ -8,7 +8,7 @@
 >
 > There is a default version of airflow shipped with each version of the chart, see the [default `values.yaml`](../../../values.yaml) for the current one.
 >
-> Many versions of airflow versions are supported by the chart, please see the [Airflow Version Support](../../..#airflow-version-support) matrix.
+> Many versions of airflow versions are supported by the chart, please see the [Airflow Version Support](../../../README.md#airflow-version-support) matrix.
```

Comment:
I think we can avoid directly linking to the README, by just changing it to `../../../#airflow-version-support` (note the slash was missing)

Reply:
Hi, I tried it but version without `README.md` redirects me and doesn't scroll to the anchor. I used Google Chrome on macOS.

**With `README.md#key-features`**
https://github.com/marekhanus/charts/blob/feature/docs/charts/airflow/docs/guides/quickstart.md

redirects to:
> https://github.com/marekhanus/charts/blob/feature/docs/charts/airflow/README.md#key-features

**Just `#key-features`**
https://github.com/marekhanus/charts/blob/feature/docs_v2/charts/airflow/docs/guides/quickstart.md

redirects to:
> https://github.com/marekhanus/charts/tree/feature/docs_v2/charts/airflow

Reply:
@marekhanus that's really interesting, I see that you are correct.

If you are wondering, the reason I was wrong was because when you are editing the page in the GitHub UI, if you use the "preview" of the page, the links work correctly with the anchor, but when the pages are actually in the repo, the anchor is removed.